src/main/ionic/peers/src/components/edit-profile-modal/edit-profile-modal.ts

```diff
@@ -0,0 +1,56 @@
+import { Component } from '@angular/core';
+import {NavController, NavParams, Platform, ViewController} from 'ionic-angular';
+import {UserService} from "../../services/user-service";
+import {AppState} from "../../states/app-state";
+import { MyProfile } from "../../pages/my-profile/my-profile";
+
+@Component({
+  selector: 'edit-profile-modal',
+  templateUrl: 'edit-profile-modal.html'
+})
+export class EditProfileModal {
+  data: any;
+  userInfo = {
+    firstName: '',
+    lastName: '',
+    level: '',
+    major: '',
+    interests: '',
```

Comment:
We should remember to change this to the tags implementation that Dan used for create account